examples/gallery/plot_AIA_limb_STEREO.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+=================================
+Drawing AIA Limb on STEREO Images
+=================================
+
+
+In this example we use a STEREO-B and an SDO image to demonstrate how to
+overplot the limb as seen by AIA on an EUVI-B image. This makes use of
+functionality added in Astropy 1.3.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+from __future__ import print_function, division
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+import sunpy.coordinates
+import sunpy.coordinates.wcs_utils
+from sunpy.net import vso
+
+
+##############################################################################
+# The first step is to download some data, we are going to get an image from
+# early 2011 when the STEREO spacecraft were roughly 90 deg seperated from the
+# Earth.
+
+stereo = (vso.attrs.Source('STEREO_B') &
+          vso.attrs.Instrument('EUVI') &
+          vso.attrs.Time('2011-01-01', '2011-01-01T00:10:00'))
+
+aia = (vso.attrs.Instrument('AIA') &
+       vso.attrs.Sample(24 * u.hour) &
+       vso.attrs.Time('2011-01-01', '2011-01-02'))
```

Comment:
What I meant to use a sdo image closer to the stereo image, instead the other way around